### PR TITLE
Feat/algorithman/localization manager changes

### PR DESCRIPTION
**Pull request by Algorithman**

### CODE REVIEW

File: JotunnLib/Managers/LocalizationManager.cs [hunk 1]
```diff
@@ -228,32 +228,28 @@ private void AddLanguageFilesFromPluginFolder()
         /// <param name="forceReplace">replace the token if it already exists</param>
         public void AddToken(string token, string value, string language = DefaultLanguage, bool forceReplace = false)
         {
-            if (token[0] != TokenFirstChar)
```
Comment (sirskunkalot) [inline JotunnLib/Managers/LocalizationManager.cs:231]: why did you strip that? for the tokens it should be fine.

**Review by paddywaan — APPROVED** [review]

**Review by raduschirliu — APPROVED** [review]

File: JotunnLib/Managers/LocalizationManager.cs (hunk 1 above)
Comment (raduschirliu) [reply JotunnLib/Managers/LocalizationManager.cs:231]: Wouldn't we want to treat single tokens in the same way that we treat an entire localization dictionary though?
If we can add a dictionary that doesn't require tokens to start with `$`, I would think that adding single tokens should behave the same way

Comment (sirskunkalot) [reply JotunnLib/Managers/LocalizationManager.cs:231]: OK, if you are sure of that change, lets add it :)

Comment (raduschirliu) [inline JotunnLib/Managers/LocalizationManager.cs:231]: Would we be able to get rid of `TokenFirstChar` entirely now? Or is it still being used somewhere

Comment (Algorithman) [reply JotunnLib/Managers/LocalizationManager.cs:231]: Still used in Skillsconfig iirc. Search for "$" (including the quotes)

**Pull request merged**